stdlib/regex.gr

```diff
@@ -4067,3 +4066,68 @@ export let replaceAll =
   ) => {
   regexReplaceHelp(rx, toSearch, replacement, true)
 }
+
+let regexSplitHelp = (rx: RegularExpression, str: String, all: Bool) => {
```

Comment:
Does this no longer have the problem specified in https://github.com/grain-lang/grain/issues/1474 ?

Reply:
It no longer uses any sort of recursion so that problem does not apply anymore